src/renderer/containers/onboard/OnboardContainer.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {Component} from "react";
+import {Background} from "../../components/Background/Background";
+import {Modal} from "../../components/Modal/Modal";
+import {Redirect} from "react-router-dom";
+import {ButtonPrimary, ButtonSecondary} from "../../components/Button/ButtonStandard";
+import {StepNavigation} from "../../components/StepNavigation/StepNavigation";
+
+const steps = [
+    {stepId: 1, stepName: "Signing key"},
+    {stepId: 2, stepName: "Withdrawal key"},
+    {stepId: 3, stepName: "Password"},
+    {stepId: 4, stepName: "Configure"},
+    {stepId: 5, stepName: "Consent"}
+];
+
+export default class OnboardContainer extends Component<{}, { isRedirectToMain: boolean }> {
+
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            isRedirectToMain: false
+        };
+
+        this.handleBack = this.handleBack.bind(this);
```

Comment:
pretty sure you don't need this as you are using arrow function which binds this by default